services/network/network.go

```diff
@@ -891,8 +895,6 @@ type PrivateEndpoint struct {
 type InterfacePropertiesFormat struct {
 	// VirtualMachine - READ-ONLY; The reference of a virtual machine.
 	VirtualMachine *SubResource `json:"virtualMachine,omitempty"`
```

Comment:
Does removing NetworkSecurityGroup break any older clients ?  @msftpgayam thoughts ?

Reply:
Since it's not being used right now, I don't think it will break the older client. It should be ok to remove a field in the sdk.